test/13031_temperature.js

```diff
@@ -27,7 +27,7 @@ describe('Temperature: ', function () {
   })
 
   it('all 130312 mappings are valid', function () {
-    var temperatureMappings = require('../n2kMappings').temperatureMappings
+    var temperatureMappings = {}
```

Comment:
This is obviously wrong but this is equivalent to what we had before here: `../n2kMappings` did not contain the key `temperatureMappings`.

The real fix is in #125.